src/Components/Components/src/LayoutView.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Components
+{
+    /// <summary>
+    /// Displays the specified content inside the specified layout and any further
+    /// nested layouts.
+    /// </summary>
+    public class LayoutView : IComponent
```

Comment:
We previously ended up naming things like `FooDisplay` instead of `FooView`. I notice this PR adds a bunch of "Views". I'm OK with whatever one we choose, but it should be consistent.

Reply:
Yes, in this PR I was specifically aiming to kill the `...Display` convention and replace it wholesale with `...View`. That's because previously we had an inconsistency (`AuthorizeView` vs `PageDisplay`), so we should pick one or the other. Of the two, the choice was pretty arbitrary but I feel good about the name `AuthorizeView` and didn't really want to change that.

Reply:
Cool, I'm happy with that resolution.